client/src/mapView.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { Map, TileLayer } from 'react-leaflet';
+import ReactModal from 'react-modal';
+import { tileSet, SFGeo, zoomLevel, mapAttribution } from '../mapconfig';
+import MarkerCollection from './markerCollection';
+
+const PARKLETS_ENDPOINT = 'https://data.sfgov.org/resource/6a7x-cttf.json';
+
+const fetchData = (url) => {
```

Comment:
this should be in it's own file and imported into here as a helper

Reply:
Isn't think also what our crawler does? Is there a reason we have this helper getting info on the client and on the server?

Reply:
@aricalves do we have an endpoint I can use for fetching data ready?  If not, I'm going to leave this in here temporarily.

Reply:
@mikehern We do not at the moment. Leave the temporary fetch there as a workaround for now until we have our backend up and running

Reply:
Solid. I still say to break it out into a module since its a dependency but not related to the react component. Separations of concerns and all

Reply:
Also you can just return the promise after all the work is done instead of declaring it to a var. Its not really needed to declare since its going to be used by an external function